### DISCUSSION TITLE
how to create an nbbo feed with different symbols?

### DISCUSSION THREAD

**Discussion by trading9991** [bmoscon/cryptofeed#712] [Q&A] [answered]
Hi,

I'm having difficulty using the NBBO feed. I am trying to adding different exchanges with different symbols into the feed. I want to add OKEx, FTX with contract 'BTC-USD-PERP' and dYdX with contract 'BTC-USD'. is it possible to add them together? Thank you. 

`f.add_nbbo([OKEx, FTX], ['BTC-USD-PERP'], nbbo_update)`
`f.add_nbbo([dYdX], ['BTC-USD], nbbo_update)`

Thanks

**Comment by bmoscon** [comment, marked as answer]
You can't - such a thing wouldnt make sense. You can only make an NBBO of the same symbols. Similarly, you need to add at least two exchanges. I can add some code so these things raise errors if they are attempted.
